wire/modules/Image/ImageSizerEngineIMagick/ImageSizerEngineIMagick.module

```diff
@@ -24,6 +24,7 @@ class ImageSizerEngineIMagick extends ImageSizerEngine {
 	 * 
 	 */
 	protected $im = null;
+	protected $imWebp = null;
```

Comment:
This should probably be commented separately. Additionally the code block above isn't particularly helpful as-is, since it doesn't explain what `$im` or `$imWebp` are actually for.